src/test/java/org/opentripplanner/middleware/TestUtils.java

```diff
@@ -1,29 +1,55 @@
 package org.opentripplanner.middleware;
 
+import org.eclipse.jetty.http.HttpStatus;
 import org.opentripplanner.middleware.auth.Auth0UserProfile;
+import org.opentripplanner.middleware.models.AbstractUser;
+import org.opentripplanner.middleware.models.ApiUser;
+import org.opentripplanner.middleware.otp.OtpDispatcherResponse;
+import org.opentripplanner.middleware.utils.FileUtils;
 import org.opentripplanner.middleware.utils.HttpUtils;
+import spark.Request;
+import spark.Response;
+import spark.Service;
 
 import java.net.URI;
 import java.net.http.HttpResponse;
 import java.util.HashMap;
 
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+import static org.opentripplanner.middleware.auth.Auth0Users.getAuth0Token;
+import static spark.Service.ignite;
+
+
 public class TestUtils {
 
     /**
-     * Returns true only if an environment variable exists and is set to "true".
+     * Prevents the mock OTP server being initialized more than once
      */
-    public static boolean getBooleanEnvVar(String var) {
-        String variable = System.getenv(var);
-        return variable != null && variable.equals("true");
-    }
+    private static boolean mockOtpServerSetUpIsDone = false;
+
+    /**
+     * Mock plan endpoint used by the mock OTP server.
+     */
+    static final String MOCK_OTP_PLAN_ENDPOINT = "otp/plan";
+
+    /**
+     * Password used to create and validate temporary Auth0 users
+     */
+    static final String TEMP_AUTH0_USER_PASSWORD = "t3mp-pa$$w0rd";
```

Comment:
I don't like the idea of storing a password in code like this, even if it's just temporary.

Reply:
Fixed in https://github.com/ibi-group/otp-middleware/pull/64/commits/6ff1fd1e10f9a19ca588b960984e4d4c68aca61d